src/libraries/System.Windows.Extensions/tests/System/Media/SoundPlayerTests.cs

```diff
@@ -462,6 +462,7 @@ public void StreamChanged_AddRemove_Success()
         [InlineData(0)]
         [InlineData(1)]
         [InlineData(2)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/41676", TestPlatforms.Browser)]
```

Comment:
this library doesn't run on Browser, you probably meant:

```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/41676", TestRuntimes.Mono)]
```

Reply:
Damn, muscle memory 😄